src/main/java/jenkins/plugins/office365connector/Office365ConnectorWebhookNotifier.java

```diff
@@ -146,20 +125,13 @@ public void sendBuildMessage(StepParameters stepParameters) {
             return;
         }
 
-        String webhookUrl = stepParameters.getWebhookUrl();
-        try {
-            WebhookJobProperty property = (WebhookJobProperty) run.getParent().getProperty(WebhookJobProperty.class);
-            if (property == null) {
-                return;
-            }
-            for (Webhook webhook : property.getWebhooks()) {
-                webhookUrl = webhook.getUrl();
-                HttpWorker worker = new HttpWorker(run.getEnvironment(listener).expand(webhookUrl), gson.toJson(card), webhook.getTimeout(), 3, listener.getLogger());
-                executorService.submit(worker);
-            }
-        } catch (Throwable error) {
-            error.printStackTrace(listener.error(String.format("Failed to notify webhook '%s'", webhookUrl)));
-            listener.getLogger().println(String.format("Failed to notify webhook '%s' - %s: %s", webhookUrl, error.getClass().getName(), error.getMessage()));
+        WebhookJobProperty property = (WebhookJobProperty) run.getParent().getProperty(WebhookJobProperty.class);
```

Comment:
hmm, this code also could be moved to ``executeWorker`` but then this refactor is getting bigger and bigger